wpeverest/everest-forms
# Enhancement - Keyboard shortcut key (Ctrl+h) for showing keyboard shortcut keys in modal.

### All Submissions:




### Changes proposed in this Pull Request:

Added keyboard shortcut key (Ctrl+h) for showing keyboard shortcut keys in modal.

### How to test the changes in this Pull Request:

1. Go to Everest Forms Builder.
2. Press keyboard shortcut Ctrl + h.
3. Verify, modal with keyboards shortcut keys, opening or not. 

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Keyboard shortcut key (Ctrl+h) for showing keyboard shortcut keys in modal.
